system-tests/test_filewriter_ignores_commands.py

```diff
@@ -0,0 +1,18 @@
+from helpers.kafkahelpers import create_producer, send_writer_command
+from time import sleep
+from subprocess import check_output
+
+# Simple system test for checking that a file writer only responds to commands given with the correct service-id.
+# Exit command tested, shouldn't matter if the file-writing has started or not.
+
+def test_ignores_commands_with_incorrect_id(docker_compose_multiple_instances):
+    producer = create_producer()
+    sleep(10)
+    send_writer_command("commands/writer-exit-single.json", producer, "TEST_writerCommandMultiple")
```

Comment:
I suggest adding some comments to make it clear what you are doing in the test: "Command only filewriter2 to exit"